src/Service.js

```diff
@@ -579,10 +586,12 @@ Service_.prototype.saveToken_ = function(token) {
 
 /**
  * Gets the token from the service's property store or cache.
+ * @param {boolean} optSkipMemory Whether to bypass the local memory cache when
```

Comment:
- `@param {boolean?}` is pretty common for optionals.
- It's pretty common to see parameters as nully in JS, so I'd write `skipMemory` instead of `optSkipMemory`. We're also not "skipping" memory, but rather the memory check. `skipMemoryCheck`?
- I find `skipCache` more understandable than `skipMemory`. Maybe `skipCacheCheck` then?
- Description: `If true, bypasses the local memory cache when fetching the token.`
- Remove `(the default is false)`

```js
 @param {boolean?} skipCacheCheck If true, bypasses the local memory cache
    when fetching the token.
```

It's not exact JSDoc, but it's pretty common to see this format.

Reply:
As mentioned below, `skipCache` doesn't work. I've used `skipMemoryCheck` instead.